src/framework/accessibility/internal/accessibilitycontroller.cpp

```diff
@@ -352,8 +356,9 @@ void AccessibilityController::triggerRevoicingOfChangedName(IAccessible* item)
         if (m_lastFocused) {
             m_lastFocused->setState(State::Focused, false);
         }
-
-        m_itemForRestoreFocus->setState(State::Focused, true);
+        if (m_itemForRestoreFocus) {
```

Comment:
please add empty lines before and after block

Reply:
Noted, and I'll make that change if we go with this type of solution.

This still isn't a good fix, though.  It solves the crash but produces very strange screen reader output as noted in https://github.com/musescore/MuseScore/issues/16437#issuecomment-1610561150.  I'd like to do better, but I am still not sure how.

But because crashing is way worse than occasionally misleading screen reader info, I'll update the PR to add the space, squash the commits, and take it out of draft status so we can consider merging for now and hopefully finding something better later.